src/Ryujinx.Common/Configuration/AppDataManager.cs

```diff
@@ -63,6 +63,18 @@ public static void Initialize(string baseDirPath)
             string userProfilePath = Path.Combine(appDataPath, DefaultBaseDir);
             string portablePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, DefaultPortableDir);
 
+            // On macOS, check for a portable directory next to the app bundle as well.
+            if (OperatingSystem.IsMacOS() && !Directory.Exists(portablePath))
+            {
+                string bundlePath = Path.GetFullPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "..", ".."));
+                // Make sure we're actually running within an app bundle.
+                if (bundlePath.EndsWith(".app"))
+                {
+                    string bundleParentPath = Path.GetFullPath(Path.Combine(bundlePath, ".."));
+                    portablePath = Path.Combine(bundleParentPath, DefaultPortableDir);
```

Comment:
```suggestion
                    portablePath = Path.GetFullPath(Path.Combine(bundlePath, "..", DefaultPortableDir));
```

Could you inline `bundleParentPath`? Since we only need it once and the `Combine` method accepts more arguments, I think it's more readable that way.